### PR TITLE
Battles: Use room.add

### PR DESCRIPTION
**Pull request by HoeenCoder**

### CODE REVIEW

File: room-battle.js [hunk 1]
```diff
@@ -496,13 +496,19 @@ class Battle {
 		switch (lines[1]) {
 		case 'update':
 			this.checkActive();
-			this.room.push(lines.slice(2));
+			lines = lines.slice(2);
+			for (let i = 0; i < lines.length; i++) {
```
Comment (KrisOnline) [inline room-battle.js]: shouldnt you use a `for...of` loop

Comment (HoeenCoder) [reply room-battle.js]: my bad

**Comment by Zarel** [comment]
Why do you guys keep using `let` instead of `const`?

**Comment by HoeenCoder** [comment]
Force of habit sorry.

**Pull request merged**